test/EVMHost.cpp

```diff
@@ -292,6 +297,8 @@ evmc::result EVMHost::call(evmc_message const& _message) noexcept
 			result.create_address = message.destination;
```

Comment:
Hmm, this seems to be broken?

Reply:
Why?

Reply:
We discussed this offline, the confusion was that the code earlier calculates the create address and assigns it to `message.destination` (before `execute`).